pkg/backup/compaction_job.go

```diff
@@ -791,26 +791,40 @@ func concludeBackupCompaction(
 	ctx context.Context,
 	execCtx sql.JobExecContext,
 	store cloud.ExternalStorage,
-	encryption *jobspb.BackupEncryptionOptions,
+	details jobspb.BackupDetails,
 	kmsEnv cloud.KMSEnv,
 	backupManifest *backuppb.BackupManifest,
 ) error {
 	backupID := uuid.MakeV4()
 	backupManifest.ID = backupID
 
 	if err := backupinfo.WriteBackupManifest(ctx, store, backupbase.BackupManifestName,
-		encryption, kmsEnv, backupManifest); err != nil {
+		details.EncryptionOptions, kmsEnv, backupManifest); err != nil {
 		return err
 	}
 	if backupinfo.WriteMetadataWithExternalSSTsEnabled.Get(&execCtx.ExecCfg().Settings.SV) {
-		if err := backupinfo.WriteMetadataWithExternalSSTs(ctx, store, encryption,
+		if err := backupinfo.WriteMetadataWithExternalSSTs(ctx, store, details.EncryptionOptions,
 			kmsEnv, backupManifest); err != nil {
 			return err
 		}
 	}
 
 	statsTable := getTableStatsForBackup(ctx, execCtx.ExecCfg().InternalDB.Executor(), backupManifest.Descriptors)
-	return backupinfo.WriteTableStatistics(ctx, store, encryption, kmsEnv, &statsTable)
+	if err := backupinfo.WriteTableStatistics(
```

Comment:
what formatter do you use?

Reply:
I'm using `crlfmt`, but it doesn't limit lines to 100 characters, so I generally just manually add those breaks if the line goes past 100 characters.